deps/install_tensorflow_1_15.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eou pipefail
+
+pip install tensorflow==1.15.0
+
+# Find out where do the tensorflow libs and sources reside
+TF_DIR="$(python -c 'import tensorflow as tf; print(tf.sysconfig.get_lib())')"
+
+# Make TF lib detectable for the ldd linker
+ln -s "$TF_DIR/libtensorflow_framework.so.1" "$TF_DIR/libtensorflow_framework.so"
```

Comment:
Why is this needed? So everyone else also needs to do this? Shouldn't it be simple to just fix the compile/ld flags at compilation to directly find the right so-file? I think this would be the better solution.

Reply:
Let me check again then - I had no success with this approach earlier.